docs/UserGuide.md

```diff
@@ -250,6 +250,33 @@ SUCCESS: Medicine has been deleted
 
 ## Search
 
+To search for a medication, you can type 'search', followed by the field you want to search for and the keyword 
+`keyword`.
+You have to include at least one of the following flags: `-n`, `-i`, `-s`, `-a` to specify the field you want to 
+search for.
+
+```
+Usage:
+    search ([-n name] [-i illness] [-s sideEffects] [-a allFields] keyword) [-h]
+Options:
```

Comment:
Would leaving a line before Options increase readability?

Reply:
resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/142/commits/415b6935cb0b940f93876e6598e7cf97ec419f0c